ibis/pandas/udf.py

```diff
@@ -542,16 +542,9 @@ def pre_execute_elementwise_udf(
     # Define an execution rule for elementwise operations on a
     # grouped Series
     nargs = len(input_type)
-    group_by_signatures = [
-        udf_signature(input_type, pin=pin, klass=SeriesGroupBy)
-        for pin in range(nargs)
-    ]
-
-    @toolz.compose(
-        *(
-            execute_node.register(ops.ElementWiseVectorizedUDF, *types)
-            for types in group_by_signatures
-        )
+
+    @execute_node.register(
+        ops.ElementWiseVectorizedUDF, *(itertools.repeat(SeriesGroupBy, nargs))
```

Comment:
Why do we not also use the object rule here for a 'scalar input catch-all'?

Reply:
Because we only need one catch all for `ops.ElementWiseVectorizedUDF`

We do not need another here because both of these register rule for `ops.ElementWiseVectorizedUDF`